src/Host/Broker/Impl/Microsoft.R.Host.Broker.csproj

```diff
@@ -23,6 +23,7 @@
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="1.1.1" />
     <PackageReference Include="System.Net.NetworkInformation" Version="4.3.0" />
     <PackageReference Include="System.Net.Requests" Version="4.3.0" />
+    <PackageReference Include="System.Net.WebSockets.Client" Version="4.3.1" />
```

Comment:
Does it need to be added to `<VSIXSourceItem>` in package as well as to Wix (remote setup) and `swix` in RTVS-pr?